hera_pspec/pspecdata.py

```diff
@@ -2789,7 +2865,22 @@ def validate_blpairs(blpairs, uvd1, uvd2, baseline_tol=1.0, verbose=True):
             if np.linalg.norm(bl1_vec - bl2_vec) >= baseline_tol:
                 raise_warning("blpair {} exceeds redundancy tolerance of {} m".format(blp, baseline_tol), verbose=verbose)
 
+
 def raise_warning(warning, verbose=True):
     '''warning function'''
     if verbose:
         print(warning)
+
+
+def _load_dsets(fnames, bls=None, pols=None, logf=None, verbose=True):
```

Comment:
Well, its a helper function only for `PSpecData` right now. Idk how you feel about dumping things into `utils`, my philosophy is its a helper function that is needed by more than one function, put it in `utils`, otherwise don't, but I'm happy to put it in `utils` if you think it makes sense

Reply:
OK, I can buy that.